# RNAseq Workshop at NMNH

These are the [materials](https://github.com/SmithsonianWorkshops/2020-01-28-NMNH-RNAseq/tree/master/Materials) used for the RNAseq course given at the Smithsonian Natural History Museum on January 28th 2020. 

**Note: This tutorial has been updated and modified from the Trinity tutorial given by Brian Haas [here](https://github.com/trinityrnaseq/KrumlovTrinityWorkshopJan2016/wiki/Home/e67c7a4ae4fe005866a56371ea29f15c79e8ccfb) for use on the Smithsonian HPC. We will also use data from [Defining the transcriptomic landscape of Candida glabrata by RNA-Seq](http://www.ncbi.nlm.nih.gov/pubmed/?term=25586221). We are very grateful that these materials have been made public by the Trinity team!**

### Workshop Outline

| Schedule  | Topic  |
|---|---|
|9:00 am |Introduction to Workshop   |
|9:15 am |Wet Lab discussion/LAB Resources  |
|10:00 am|Informatics Introduction   |
|10:15 am|BREAK  |
|10:30 am|Evaluate Read Quality (FASTQC)  |
|11:00 am|Trimming & Data refinement (Trimgalore)  |
|11:30 am|De Novo Transcriptome Assembly (Trinity)  |
|12:00 pm|Lunch  |
|1:15 pm |QA/QC of Assembly  |
|1:45 pm |Transcript Quantification  |
|2:30 pm |Differential Expression Analysis  |
|3:00 pm |BREAK  |
|3:15 pm |Differential Expression Analysis (Continued)  |

### Useful Links

Etherpad link: https://pad.carpentries.org/2020-01-28-NMNH-RNAseq

Interested in future workshops? Please [keep in touch.](keeping_in_touch.md)

### PRECOURSE SURVEY: https://forms.gle/cGz68tMjbtz5ZVA67
